<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>606</width>
    <height>531</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>fconvertor</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_input">
          <property name="text">
           <string>输入视频:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_input"/>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_input">
          <property name="text">
           <string>选择</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTextBrowser" name="textBrowser_input"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="label_size">
              <property name="text">
               <string>输出视频分辨率:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_size"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="label_output">
              <property name="text">
               <string>输出视频:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_output"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QPushButton" name="pushButton_2">
              <property name="text">
               <string>转换</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton">
              <property name="text">
               <string>选择</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTextBrowser" name="textBrowser_output"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QProgressBar" name="progressBar_convert">
        <property name="value">
         <number>24</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
   <zorder>label_input</zorder>
   <zorder>label_output</zorder>
   <zorder>lineEdit_input</zorder>
   <zorder>pushButton_input</zorder>
   <zorder>textBrowser_input</zorder>
   <zorder>label_size</zorder>
   <zorder>lineEdit_size</zorder>
   <zorder>lineEdit_output</zorder>
   <zorder>pushButton</zorder>
   <zorder>textBrowser_output</zorder>
   <zorder>pushButton_2</zorder>
   <zorder>progressBar_convert</zorder>
   <zorder></zorder>
   <zorder></zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>606</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
